galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -63,6 +63,10 @@ def get_pulp(self, distro):
                 'name': repo.name,
                 'description': repo.description,
                 'pulp_created': repo.pulp_created,
+                'last_sync_task': _get_last_sync_task(repo),
+                'pulp_labels': {
+                    label.key: label.value for label in repo.pulp_labels.all()
```

Comment:
@newswangerd I was unsure if labels here are from `repository` or `distribution` both have labels.

Reply:
@rochacbruno you can leave this on the repository. Lets add it to the distribution as well.